com.hack_computer.Clippy.Extension.json

```diff
@@ -2,7 +2,7 @@
     "app-id": "com.hack_computer.Clippy.Extension",
     "runtime": "com.hack_computer.Clubhouse",
     "runtime-version": "beta",
-    "sdk": "org.gnome.Sdk//41",
+    "sdk": "org.gnome.Sdk//43",
```

Comment:
the app runtime is still 41.

See also https://github.com/flathub/com.hack_computer.Clubhouse/pull/11

Reply:
Ah thanks